test/Microsoft.AspNet.Mvc.FunctionalTests/LocalizationTest.cs

```diff
@@ -71,5 +74,100 @@ public static IEnumerable<object[]> LocalizationData
             // Assert
             Assert.Equal(expected, body.Trim());
         }
+
+        public static IEnumerable<object[]> LocalizationResourceData
+        {
+            get
+            {
+                var expected1 =
+ @"My ASP.NET Application
+
+Hello there!!
+Learn More";
+
+                yield return new[] {"en-GB", expected1 };
+
+                var expected2 =
+ @"Mon application ASP.NET
+
+Bonjour!
+apprendre Encore Plus";
+
+//Coreclr just reads enu resources. Hence the conditional if
```

Comment:
Is this a bug? Known limitation? Just temporary? Do we need a bug to track some later work? If we need to log a bug, can reference the bug from the comment here.